Koha/Plugin/EDS/bootstrap/includes/eds-facets.inc

```diff
@@ -1,6 +1,6 @@
 <div id="search-facets">
-    <h4><a href="#" class="menu-collapse-toggle">[% EDSLANG.eds_facets_INC_refine_search %]</a></h4>
-    <ul class="menu-collapse">
+        <h2><a href="#" class="menu-collapse-toggle">Refine your search</a></h2>
```

Comment:
It looks like Refine Your Search has been hard coded instead of using the language file from the template to handle translations.  Is this intentional?

Reply:
Review comments fixed.

Reply:
Review comments fixed.